packages/shared/src/server/llm/fetchLLMCompletion.ts

```diff
@@ -69,8 +97,35 @@ export async function fetchLLMCompletion(
     baseURL,
     maxRetries,
     config,
+    traceParams,
   } = params;
 
+  let finalCallbacks: BaseCallbackHandler[] | undefined = callbacks ?? [];
+  let processTracedEvents: ProcessTracedEvents = () => Promise.resolve();
+
+  if (traceParams) {
```

Comment:
in which cases is this set?

Reply:
whenever we would like to trace a fetchLLMCompletion call. We don't trace by default, only if traceParams are provided